github-actions/lock-closed/src/main.ts

```diff
@@ -0,0 +1,78 @@
+import * as core from '@actions/core';
+import * as github from '@actions/github';
+
+async function lockIssue(client: github.GitHub, issue: number, message?: string): Promise<void> {
+  // Add a comment before locking
+  if (message) {
+    await client.issues.createComment({
+      owner: github.context.repo.owner,
+      repo: github.context.repo.repo,
+      issue_number: issue,
+      body: message,
+    });
+  }
+
+  // Actually lock the issue
+  await client.issues.lock({
+    owner: github.context.repo.owner,
+    repo: github.context.repo.repo,
+    issue_number: issue,
+  });
+}
+
+async function run(): Promise<void> {
+  try {
+    // NOTE: `days` and `message` must not be changed without dev-rel and dev-infra concurrence
+
+    // Fixed amount of days a closed issue must be inactive before being locked
+    const days = 30;
+    // Standardized Angular Team message for locking issues
+    const message =
```

Comment:
Would it be better to use [`commonTags.stripIndent`](https://github.com/declandewet/common-tags#stripindent) and move this to a template string?

This would also allow us to include the link as a variable a little easier.

```ts
// NOTE: `policyUrl`, `days` and `message` must not be changed without dev-rel and dev-infra concurrence

const policyUrl = `https://github.com/angular/angular/{sha}/docs/GITHUB_PROCESS.md#automatic-conversation-locking`;
// Fixed amount of days a closed issue must be inactive before being locked
const days = 30;
// Standardized Angular Team message for locking issues
const message = stripIndent`
  This issue has been automatically locked due to inactivity.
  Please file a new issue if you are encountering a similar or related problem.

  Read more about our [automatic conversation locking policy](${policyUrl})

  _This action has been performed automatically by a bot._`;
```

Reply:
Added the policyUrl line as a template string.

I usually don't add in `common-tags` unless its needed several times in the codebase.  If you prefer its use, I can add it in though.

Reply:
That's fine with me